summariziation_example.ipynb

```diff
@@ -15,7 +15,7 @@
   },
```

Comment:
1. Add 2 new columns for [cost/1MT In ($)] [cost/1MT out ($)] - note it’s cost 1 million tokens (the cost per 1K token is too small). This cost shows the price list we used to calculate the actual cost column.  
2. Change latency column to be in seconds so it should display: 22.87 instead of 22869.8900
3. Make sure to submit the notebook without any outputs (clear outputs then commit) - it’s easy to forget.
Add tooltips to all columns except model name - put some basic text that we could change later.
4. rename `cost ($)` to `Total  cost (3)`.